src/plugins/cloudinary-analytics/index.js

```diff
@@ -0,0 +1,22 @@
+import { trackVideoPlayer } from '../../components/cld-analytics/cld-analytics';
```

Comment:
I'd put the whole thing inside this `cloudinary-analytics` plugin. why do we split it into a plugin AND a component?

Reply:
The whole idea is to support `cloudinary-video-player` for now but in the future wa want to extract this part as separated package so it can be used with any other video player like this:

```
trackVideoPlayer(document.querySelector('video'), {
  videoPublicId: 'some-video',
  cloudName: 'demo',
});
```